homeassistant/components/thermostat/zwave.py

```diff
@@ -109,6 +134,24 @@ def target_temperature(self):
     def set_temperature(self, temperature):
         """Set new target temperature."""
         # set point
+        if self._zxt_120 == 1:
```

Comment:
Yeah I agree that a new component makes sense here. A thermostat should not be aware of swinging modes.